.kokoro/functions/speech-to-speech.cfg

```diff
@@ -0,0 +1,25 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Set the folder in which the tests are run
+env_vars: {
+    key: "PROJECT"
+    value: "functions/speech-to-speech"
+}
+
+# Tell the trampoline which build file to use.
+env_vars: {
+    key: "TRAMPOLINE_BUILD_FILE"
+    value: "github/nodejs-docs-samples/.kokoro/build.sh"
+}
+
+# Environment values for tests that Kokoro doesn't provide natively
+env_vars: {
```

Comment:
Have you created an (internal) Kokoro config as well? If not, please create it and add @fhinkel (franzih@) as a reviewer.

Ping me or @fhinkel if you need more info on this.

Reply:
Done, marking as resolved.